### DISCUSSION TITLE
Fail to set up page locally - Deployment works

### DISCUSSION THREAD

**Discussion by LabrosV** [alshedivat/al-folio#1053] [Q&A] [answered]
Hi everyone,

I tried to follow the instructions from the recommended video (https://www.youtube.com/watch?v=g6AJ9qPPoyc).

I had an old version of Ruby and I used rbenv (https://github.com/rbenv/rbenv) to load the version 2.7.7 following the instructions from here (https://stackoverflow.com/questions/38194032/how-can-i-update-ruby-version-2-0-0-to-the-latest-version-in-mac-os-x-v10-10-yo). 
Gems were created successfully but when I ran the command "bundle exec jekyll serve" and jekyll (in general) the command crashes with the message complaining about /usr/lib/system/libsystem_platform.dylib  /usr/lib/dyld.

Do you think that this is because of my MacOS version (catalina)? I did homebrew update without upgrading it.

Labros

**Comment by pourmand1376** [comment, marked as answer]
Hi, I recommend testing with docker installation. Most probably, you wouldn't have any problems with that.
